app/domain/auth/templates/login.tpl.php

```diff
@@ -33,6 +33,14 @@
         <div class="">
             <input type="submit" name="login" value="<?php echo $this->language->__("buttons.login"); ?>" class="btn btn-primary"/>
         </div>
+        <?php if($this->get('oidcEnabled')) { ?>
```

Comment:
Can we move this to the top to afterRegContent? 
The reason I ask is we have the Google Auth plugin built by @m8than 
He added the button to the top (https://github.com/m8than/googleSSO-leantime/blob/f0c44fa29c6817b30ed3427ba1ac966d7cad521c/register.php#L107) and we should keep the login providers in the same place.